tests/workbenches/utils.py

```diff
@@ -0,0 +1,61 @@
+#
+# Copyright Skodjob authors.
+# License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+#
+from __future__ import annotations
+
+import io
+import pathlib
+from contextlib import contextmanager
+from typing import Generator
+
+from pytest_testconfig import config as py_config
+
+from kubernetes.dynamic import DynamicClient, Resource, ResourceInstance
+
+from ocp_resources.route import Route
+from ocp_resources.notebook import Notebook
+
+from utilities.constants import INTERNAL_IMAGE_REGISTRY_PATH
+
+
+def get_notebook_image(image_name: str, image_tag: str) -> str:
+    controllers_namespace = py_config["applications_namespace"]
```

Comment:
Simple way to do this will be:
```
image_name = "jupyter-minimal-notebook" if py_config.get("distribution") == "upstream" else "s2i-minimal-notebook"
return f"{INTERNAL_IMAGE_REGISTRY_PATH}/{py_config["applications_namespace"]}/{image_name}:{image_tag}"
```

And you can drop `image_name` from the function args

Reply:
I'm not bothered by the extra argument, and i think that at callsite

```
notebook_image: str = get_notebook_image(image_name="jupyter-minimal-notebook", image_tag="2024.2")
```

makes more sense than if image_name parameter was omitted.

I could (eventually) have enum for the possible image names...

Reply:
Up to you, I would remove this function and just set the path in the test where it's needed. (only used in one place) and introduce more complicity without any reason.

Reply:
+1 to that; used once and should be set where used

Reply:
done